core/cmd/ocr2vrf_configure_commands.go

```diff
@@ -206,11 +213,26 @@ func (cli *Client) ConfigureOCR2VRFNode(c *clipkg.Context) (*SetupOCR2VRFNodePay
 			}
 
 			sendingKeys = append(sendingKeys, k.Address.String())
+			sendingKeysAddresses = append(sendingKeysAddresses, k.Address)
+		}
+
+		// We have to set the authorized senders on-chain here, otherwise the job spawner will fail as the
+		// forwarder will not be recognized.
+		ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+		defer cancel()
+		f, err := authorized_forwarder.NewAuthorizedForwarder(common.HexToAddress(forwarderAddress), Ec)
+		tx, err := f.SetAuthorizedSenders(Owner, sendingKeysAddresses)
+		if err != nil {
+			return nil, err
+		}
+		_, err = bind.WaitMined(ctx, Ec, tx)
+		if err != nil {
+			return nil, err
```

Comment:
Seems odd to do this here - we can't do this in the scripts instead?

Reply:
Agreed. If we do this before the node setup, we don't have the extra sending keys for the node yet. If we do this after the node setup, the job creation fails as the forwarder cannot be found for the EOAs (they are not yet added as authorized senders). We could split up this part of the script into two, but that could be a can of worms.

Reply:
Ah I see. I guess this is a necessary evil then. Should be okay since it's just for local setup.